### PR TITLE
Add suplink icon

### PR DESCRIPTION
**Pull request by islam3zzat**
#### Summary

In this PR we add the Suplink icon.

suplink

### CODE REVIEW

File: src/components/icons/index.js [hunk 1]
```diff
@@ -329,6 +330,7 @@ export const SpeedometerIcon = createStyledIcon(
 export const SplitIcon = createStyledIcon(OrigSplitIcon, 'SplitIcon');
 export const StackIcon = createStyledIcon(OrigStackIcon, 'StackIcon');
 export const StarIcon = createStyledIcon(OrigStarIcon, 'StarIcon');
+export const SuplinkIcon = createStyledIcon(OrigSuplinkIcon, 'SuplinkIcon');
```
Comment (tdeekens) [inline src/components/icons/index.js]: Wait. Isn't it 'SubLink`?

Comment (islam3zzat) [reply src/components/icons/index.js]: I knew something was off 🙈

File: src/components/icons/index.js [hunk 2]
```diff
@@ -330,7 +330,7 @@ export const SpeedometerIcon = createStyledIcon(
 export const SplitIcon = createStyledIcon(OrigSplitIcon, 'SplitIcon');
 export const StackIcon = createStyledIcon(OrigStackIcon, 'StackIcon');
 export const StarIcon = createStyledIcon(OrigStarIcon, 'StarIcon');
-export const SuplinkIcon = createStyledIcon(OrigSuplinkIcon, 'SuplinkIcon');
+export const SubLinkIcon = createStyledIcon(OrigSubLinkIcon, 'SubLinkIcon');
```
Comment (tdeekens) [inline src/components/icons/index.js]: I am not sure I 100% like the name. We try to name by "what is is" not "what it's for". So maybe this should be SubArrow? Sorry. @montezume should say.

Comment (islam3zzat) [reply src/components/icons/index.js]: it was `sublink` (actually `suplink`) from the SVG

Comment (islam3zzat) [reply src/components/icons/index.js]: image

Comment (islam3zzat) [reply src/components/icons/index.js]: image

**Comment by islam3zzat** [comment]
`SubdirectoryArrow` sounds good, `Subdirectory` already indicates that it's going down a level.
WDYT?

**Comment by montezume** [comment]
Yeah I think SubdirectoryArrow 🤷‍♂

**Pull request merged**